node/src/chain_spec.rs

```diff
@@ -187,6 +187,7 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
 }
 
 /// To be used when building new testnet chain-spec
+#[allow(dead_code)]
```

Comment:
i saw your point and I was the one that requested to leave it but now I prefer to remove it at all.

Anyway, feel free to leave it you prefer.

Reply:
Yes probably even better to fully remove it. In case of need we could just navigate back in the repo history.

Reply:
Oh sorry, I was late few seconds with my last push.
So right now we still have that dead code, no problem.